k8s/postgresql/README.md

```diff
@@ -231,7 +231,7 @@ helm template chart/postgresql \
   --name $APP_INSTANCE_NAME \
   --namespace $NAMESPACE \
   --set "postgresql.serviceAccount=$POSTGRESQL_SERVICE_ACCOUNT" \
-  --set "postgresql.image=$IMAGE_POSTGRESQL" \
+  --set "postgresql.image.full=$IMAGE_POSTGRESQL" \
```

Comment:
- On L138, craft new instructions for the user to locate the exact version and TAG that they would like to use. For example, the user can navigate to [the deployer gcr](https://pantheon.corp.google.com/gcr/images/cloud-marketplace/GLOBAL/google/postgresql/deployer?gcrImageListsize=30) and pick a specific release tag such as `9.6.17-20200311-092102`. Such value should be the value of the `TAG` environment variable.
- All `IMAGE_*` variables should use the ones under `.../google/postgresql` and should not include `${TAG}`. For example, the exporter image should be `marketplace.gcr.io/google/postgresql/exporter`.
- In the `helm template` command, each image should have two variables. For example, exporter image should have the following:
```
--set "exporter.image.repo=$IMAGE_POSGRESQL_EXPORTER"
--set "exporter.image.tag=$TAG"
```

Please verify that the command actually works.

Reply:
Fixed